shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2718,32 +2747,108 @@ \subsection{Reward Splitting}
+through ignoring their blocks, or even by mounting DoS attacks on their nodes).
+However, it does not incentivise pool operators to ensure that their pool
+performs well (i.e., produces most of the blocks it is eligible to create). This
+invites freeriders, pools that rely on other pools to maintain the system, and
+collect rewards without doing any work.
+
+For Cardano, we will stay close to the scheme of~\citep{bkks2018}, but we will
+also take into account the \emph{performance} of a pool. Unfortunately, tracking
+the performance of a pool is not trivial in protocols without a public leader
+schedule, like Ouroboros Praos.
+
+In order to get a handle on the performance of a pool, let us consider the
+fraction \(\beta\) of all blocks within a given epoch that the pool created.
+
+There are multiple factors that influence \(\beta\): the stake of the pool, how
+well it performs, and some randomness due to the leader election and the overall
+performance of the other pools and the network. We can write\footnote{We assume
+  a linear relation between stake and the number of blocks that a pool is
+  eligible to create. Strictly speaking, this is not the case for Ouroboros
+  Praos. However, the error when linearising the leader election function is
+  small, particularly for the range of parameters we are considering for
+  Cardano.}
+\[\beta = \sigma p r_e \,,\]
+where
+\begin{enumerate}
+\item[\(\sigma\)] is the relative stake \(\sigma\) of the pool (the fraction of
+  the absolute stake that the pool controls),
+\item[\(p\)] is the performance \(p\) of the pool, i.e. the fraction
+  \[
+  p = \frac{n}{\max(N,1)}
+  \]
+  of the number \(n\) blocks it successfully added to the chain and the number
+  \(N\) of slots it was elected as a leader, and
+\item[\(r_e\)] is a factor that captures the relation between the relative stake
+  \(\sigma\) of the pool, the number \(N\) of slots it is elected as a leader,
+  and the total number \(\overline{N}\) of blocks that were added to the chain during
+  the epoch. To be precise, we have
+  \[r_e = \frac{N}{\sigma \overline{N}}\,.\]
+  The factor \(r_e\) captures all the random influences on \(\beta\): the
+  randomness in the leader election that influences \(n\), and the randomness
+  both from the leader election and the bunch of random influences on
+  \(\overline{N}\) (leader election, forks, performance of the network and other
+  pools).
+\end{enumerate}
+
+If we insert the definitions of \(p\) and \(r_e\) into \(\beta\), we find that
+\[
+\beta = \sigma \frac{n}{\max(N,1)} \frac{N}{\sigma \overline{N}}
+= \frac{n}{\overline{N}}\,,
+\]
+which is indeed the fraction of blocks produced by the pool\footnote{When
+  cancelling \(N\) and \(\max(N,1)\), note that for \(N=0\), also \(n=0\), and
+  the equation trivially holds.}.
+
+In Ouroboros Praos, we can observe \(\sigma\), \(n\), and \(\overline{N}\), but
+we do not have a direct handle on any of \(N\), \(r_e\), or \(p\). From the
```

Comment:
Not "having a direct handle on ..." means "we cannot observe ..."?

Reply:
Yes. Should I add a clarification?

Reply:
Well, I like things to be as explicit as possible, but some more "colorful" prose is often necessary. It was just for my understanding anyway :)